.github/workflows/build.yml

```diff
@@ -416,6 +416,10 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
+      - # === Install State Tool ===
+        name: Install State Tool
+        uses: ActiveState/setup-state-tool@v1
```

Comment:
Since the workflow for generating the version list file is pretty dependent on CI I'd say let's move the scripts from the activestate.yaml into the build.yml, so we don't need to install the State Tool and thus slow down deployments.

Reply:
State tool is required for deployments. It's not just for the new feature. If you look at the build.yml file, you'll see a lot of other uses of ST (e.g. `state run install-deps`, `state run preprocess -v`, `state run deploy-updates`, etc.)

I added this block because it was missed in #2784.

Reply:
Gotcha, ok no worries that works then. I thought we were introducing it just for this story.